tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py27,py34
+skipsdist = True
+toxworkdir=/tmp
```

Comment:
Since this is not preserved it will have to be recreated whenever it runs from the command line. Is it possible to make this an environment variable and set it to `/tmp` for `travis.yml`? I found this: https://bitbucket.org/hpk42/tox/issues/20/command-line-option-for-overriding